features/s3/postObject.feature

```diff
@@ -0,0 +1,18 @@
+@s3 @integ
+Feature: POST object uploads
+
+    Scenario: Upload an object via POST object with Signature V2
+        Given I have an s3 client and I have a file
+        When I create a POST object in SigV2 with:
+        |key              |value                |
+        |condition        |{"acl":"public-read"}|
+        And I make a HTTP POST request
+        Then a file is uploaded and its filename should be "file.ext"
+
+    Scenario: Upload an object via POST object with Signature V4
+        Given I have an s3 client and I have a file
+        When I create a POST object in SigV4 with:
```

Comment:
Both tests should be more fleshed out. It's still not clear from this cucumber test what inputs a PostObject is expecting. Maybe a separate step should add the form fields?